log4j-core/src/main/java/org/apache/logging/log4j/core/config/ConfigurationFactory.java

```diff
@@ -121,6 +121,11 @@ protected boolean isActive() {
         return true;
     }
 
+    @Deprecated(since = "3.0.0", forRemoval = true)
+    public static ConfigurationFactory getInstance() {
+        return LoggerContext.getContext(false).getInstanceFactory().getInstance(KEY);
+    }
+
```

Comment:
Not sure why this is required. I can not find any usage in this PR.

Reply:
`Log4J2LoggingSystem` of Spring Boot uses this.